util/cbmem/cbmem.c

```diff
@@ -384,15 +413,22 @@ static int parse_cbtable(u64 address, size_t table_size)
 			continue;
 		}
 
+		mismatched_endianness = (lbh->header_bytes >= 0x01000000);
```

Comment:
This seems fragile if for any reason header grows up to >=256 bytes. Maybe compare it with `sizeof(struct lb_header)`, both normal and `bswap`-ped? This could additionally protect against different versions of this structure, although I expect the signature to change in that case.